examples/notify_rust.rs

```diff
@@ -0,0 +1,432 @@
+#![cfg_attr(
+    all(not(debug_assertions), target_os = "windows"),
+    windows_subsystem = "windows"
+)]
+
+use std::thread;
+
+use freya::prelude::*;
+use freya_router::prelude::*;
+use notify_rust::{Hint, Notification};
+
+fn main() {
+    launch_with_props(app, "Router Example", (550.0, 400.0));
```

Comment:
Not your fault, I changed the name of this method yesterday :D
```suggestion
    launch_with_params(app, "Router Example", (550.0, 400.0));
```

Reply:
#### Yeah, I noticed that was changed after I made the PR hehe.
#### So, I'm not sure what's the best way to resolve this. At first I tried using #[cfg(target_os = ...)] rules like in the original notify-rust examples to include the problematic examples conditionally, but then errors with the formatting started showing up, so in the end I just removed all the examples that were Linux/Unix specific.
#### Now, there's an error on the Ubuntu build about a "blocking" module in the zbus crate. That, I don't know what to do about :sweat_smile:

Reply:
Excellent! Thank you so much. Let me know if there is anything I can help with.